vslib/src/SwitchConfig.cpp

```diff
@@ -18,6 +19,35 @@ SwitchConfig::SwitchConfig():
 }
 
 
+bool SwitchConfig::parseSaiSwitchType(
+        _In_ const char* saiSwitchTypeStr,
+        _Out_ sai_switch_type_t& saiSwitchType)
+{
+    SWSS_LOG_ENTER();
+
+    std::string st = (saiSwitchTypeStr == NULL) ? "unknown" : saiSwitchTypeStr;
+
+    if (st == SAI_VALUE_SAI_SWITCH_TYPE_NPU)
```

Comment:
same here, why this is needed, this is switch_create attribute

Reply:
So that we know chip function and can avoid launch of fdb thread, which is not valid for phy.